pkg/api/customization/project/project_actions.go

```diff
@@ -0,0 +1,147 @@
+package project
+
+import (
+	"fmt"
+
+	"net/http"
+
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/rancher/norman/parse"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	managementschema "github.com/rancher/types/apis/management.cattle.io/v3/schema"
+	"github.com/rancher/types/client/management/v3"
+)
+
+func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
+	resource.AddAction(apiContext, "setpodsecuritypolicytemplate")
+}
+
+type Handler struct {
+	ProjectClient v3.ProjectInterface
+}
+
+func (h *Handler) Actions(actionName string, action *types.Action, apiContext *types.APIContext) error {
+	switch actionName {
+	case "setpodsecuritypolicytemplate":
+		return h.setPodSecurityPolicyTemplate(actionName, action, apiContext)
+	}
+
+	return errors.Errorf("bad action %v", actionName)
```

Comment:
No one should ever see this error message, but I like "unrecognized action" better than "bad Action"